src/server/templates/typescript.ts

```diff
@@ -78,8 +84,8 @@ export interface Database {
                     (table) => `${JSON.stringify(table.name)}: {
                   Row: {
                     ${[
-                      ...table.columns
-                        .sort(({ name: a }, { name: b }) => a.localeCompare(b))
+                      ...schemaColumns
+                        .filter((column) => column.table_id === table.id)
```

Comment:
Yeah, makes sense

Reply:
In case there are a lot of columns, filtering for every table may take a pretty long time (been there) - so it might make sense to map columns by table once, and then just use the table id as key for access.

This significantly improves speed with lots of tables + columns (it's crazy how slow these filters inside loops can get)